docs/AutomationReference.md

```diff
@@ -223,42 +234,43 @@ example below):
 
 #### Example
 ```C#
-    using System;
-    using System.Collections.Generic;
-    using Axe.Windows.Automation;
-
-    namespace AxeWindowsDemo
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
```

Comment:
We should ensure that our sample comples. I believe the `First()` method requires System.Linq